src/pages/Anix/main.ts

```diff
@@ -0,0 +1,119 @@
+import { pageInterface } from '../pageInterface';
+import { SyncPage } from '../syncPage';
+
+export const Anix: pageInterface = {
+  domain: 'https://anix.to',
+  languages: ['English'],
+  name: 'Anix',
+  database: '9anime',
+  type: 'anime',
+  isSyncPage(url: string): boolean {
+    return true;
+  },
+  sync: {
+    getTitle(url: string): string {
+      return j.$('h1.ani-name').text().trim();
+    },
+    getIdentifier(url: string): string {
+      const anime = url.split('/')[4];
+      const animeWords = anime.split('-');
+      return animeWords[animeWords.length - 1];
+    },
+    getOverviewUrl(url: string): string {
+      return utils.absoluteLink(j.$(episodeLinkButtonsQuery).first().attr('href'), Anix.domain);
+    },
+    getEpisode(url: string): number {
+      return parseInt(j.$(selectedEpisodeQuery).text());
+    },
+    nextEpUrl(url: string): string | undefined {
+      return j.$(selectedEpisodeQuery).parent('div').next().find('a').attr('href');
+    },
+    uiSelector(selector) {
+      j.$('#ani-player-section').before(j.html(selector));
+    },
+  },
+  overview: {
+    getTitle(url) {
+      return '';
+    },
+    getIdentifier(url) {
+      return '';
+    },
+    uiSelector(selector) {
+      // no Ui
+    },
+    list: {
+      offsetHandler: false,
+      elementsSelector() {
+        return j.$(episodeLinkButtonsQuery).not('[style*="display: none"])');
```

Comment:
Does not seem to work.
You could just use string conternation `episodeLinkButtonsQuery + ' :not([style*="display: none"])'`
![image](https://github.com/MALSync/MALSync/assets/12599334/6971d2ba-6966-4e3f-b68e-c76b8f24a5f9)